### ISSUE TITLE
Video always freezes and then lags behind the audio

### ISSUE THREAD

**Issue by cr0op** [jasongdove/ErsatzTV#1065]
Is there a video setting for the channel I can fix to prevent this from happening again? Like maybe changing the ffmpeg profile somehow? I tried switching the channel mode from MPEG-TS to HLS Segmenter but it still happens every time.

**Comment by jasongdove** [comment]
Can you give some details?

- ErsatzTV version (found below the left menu)
- Hardware acceleration method, or even better screenshot ffmpeg profile
- Client app

**Comment by cr0op** [comment]
How do you install hardware acceleration onto ersatztv?

**Comment by jasongdove** [comment]
The bundled ffmpeg contains all required codecs. If you have an intel igpu, select `Qsv` in the ffmpeg profile hardware acceleration setting, otherwise if you have an nvidia card, select `Nvenc`.

**Comment by cr0op** [comment]
I have AMD on my windows pc, does vaapi work?

**Comment by cr0op** [comment]
mostly seems to be working now. thanks for the tips!

**Comment by jasongdove** [comment]
Yes, AMF is your only option on Windows with and AMD GPU. It's limited to encoders (no decoders or filters) but I'm glad to hear it's helping a bit.

**Comment by cr0op** [comment]
just switched to hevc and it's working even better i think. maybe this'll fix that colorspace error i've been having (I think someone else mentioned it).
